Fitness/Views/ProBioPageViewController.swift

```diff
@@ -0,0 +1,313 @@
+
+        bioExpertiseDivider = UIView()
+        bioExpertiseDivider.backgroundColor = UIColor.fitnessMutedGreen
+        scrollView.addSubview(bioExpertiseDivider)
+        
+        expertiseTitleLabel = UILabel()
+        expertiseTitleLabel.textColor = .black
+        expertiseTitleLabel.font = ._16MontserratMedium
+        expertiseTitleLabel.text = "EXPERTISE"
+        expertiseTitleLabel.textAlignment = .center
+        expertiseTitleLabel.sizeToFit()
+        scrollView.addSubview(expertiseTitleLabel)
+        
+        expertiseTextView = UITextView()
+        expertiseTextView.textColor = UIColor.fitnessBlack
+        expertiseTextView.font = ._14MontserratLight
+        expertiseTextView.sizeToFit()
+        expertiseTextView.isScrollEnabled = false
+        expertiseTextView.textAlignment = .center
+        expertiseTextView.isSelectable = false
+        scrollView.addSubview(expertiseTextView)
+        
+        expertiseRoutinesDivider = UIView()
+        expertiseRoutinesDivider.backgroundColor = UIColor.fitnessMutedGreen
+        scrollView.addSubview(expertiseRoutinesDivider)
+        
+        routinesTitleLabel = UILabel()
+        routinesTitleLabel.textColor = .black
+        routinesTitleLabel.font = ._16MontserratMedium
+        routinesTitleLabel.text = "SUGGESTED ROUTINES"
+        routinesTitleLabel.textAlignment = .center
+        routinesTitleLabel.sizeToFit()
+        scrollView.addSubview(routinesTitleLabel)
+        
+        let flowLayout = UICollectionViewFlowLayout()
+        flowLayout.minimumInteritemSpacing = CGFloat(personalLinksInteritemSpacing)
+        flowLayout.minimumLineSpacing = 0
+        flowLayout.scrollDirection = .vertical
+        
+        linksTitleLabel = UILabel()
+        linksTitleLabel.text = "FOLLOW ME"
+        linksTitleLabel.font = ._12MontserratBold
+        linksTitleLabel.textColor = .black
+        linksTitleLabel.textAlignment = .center
+        linksTitleLabel.sizeToFit()
+        scrollView.addSubview(linksTitleLabel)
+        
+        linksCollectionView = UICollectionView(frame: .zero, collectionViewLayout: flowLayout)
+        linksCollectionView.delegate = self
+        linksCollectionView.dataSource = self
+        linksCollectionView.backgroundColor = .white
+        linksCollectionView.register(PersonalLinkCollectionViewCell.self, forCellWithReuseIdentifier: PersonalLinkCollectionViewCell.identifier)
+        scrollView.addSubview(linksCollectionView)
+        
+        backButton = UIButton()
+        backButton.setImage(UIImage(named: "back-arrow"), for: .normal)
+        backButton.sizeToFit()
+        backButton.addTarget(self, action: #selector(self.back), for: .touchUpInside)
+        view.addSubview(backButton)
+        view.bringSubviewToFront(backButton)
+        
+        setPro()
+        setupConstraints()
+    }
+    
+    @objc func back() {
+        navigationController?.popViewController(animated: true)
+    }
+    
+    func setPro() {
```

Comment:
can this function name be changed? `setPro` is not very descriptive

Reply:
I think pro is literally their terminology for professionals and is used quite abundantly throughout the code.

Reply:
Yes, we call the "influencers" Pros